### PR TITLE
Fix Install-Module error 'Could not compare 6.0.0-alpha to 5.0'. …

### PR DESCRIPTION
**Pull request by bmanikm**
…Added workaround fix in PowerShellGet 
Workarounds issues #1618

### CODE REVIEW

File: src/Modules/Shared/PowerShellGet/PSModule.psm1 [hunk 1]
```diff
@@ -13008,8 +13008,8 @@ function Update-ModuleManifest
     {
         #DscResourcesToExport field is not available in PowerShell version lower than 5.0
         
-        if  (($PSVersionTable.PSVersion -lt [Version]"5.0") -or ($PowerShellVersion -and $PowerShellVersion -lt [Version]"5.0") `
-             -or (-not $PowerShellVersion -and $moduleInfo.PowerShellVersion -and $moduleInfo.PowerShellVersion -lt [Version]"5.0") `
+        if  (($PSVersionTable.PSVersion -lt '5.0.0') -or ($PowerShellVersion -and $PowerShellVersion -lt '5.0') `
```
Comment (quoctruong) [inline src/Modules/Shared/PowerShellGet/PSModule.psm1]: should it be -lt '5.0.0' too?

**Comment by aarongustafson** [comment]
Please merge soon :-)

File: src/Modules/Shared/PowerShellGet/PSModule.psm1 [hunk 2]
```diff
@@ -13974,7 +13974,7 @@ function ValidateAndGet-AuthenticodeSignature
             # Skip the PSGetModuleInfo.xml and ModuleName.cat files in the catalog validation
             $TestFileCatalogResult = Microsoft.PowerShell.Security\Test-FileCatalog -Path $ModuleBasePath `
                                                                                     -CatalogFilePath $CatalogFilePath `
-                                                                                    -FilesToSkip $script:PSGetItemInfoFileName,'*.cat' `
+                                                                                    -FilesToSkip $script:PSGetItemInfoFileName,'*.cat','*.nupkg','*.nuspec' `
```
Comment (lzybkr) [inline src/Modules/Shared/PowerShellGet/PSModule.psm1]: Is this line part of the workaround?  If not, maybe it should be in another commit.

File: src/Modules/Shared/PowerShellGet/PSModule.psm1 (hunk 1 above)
Comment (bmanikm) [inline src/Modules/Shared/PowerShellGet/PSModule.psm1]: @quoctruong, not required for now, $PowerShellVersion is still of Version type and not SemanticVersion. We will have to make lot of changes to support SemanticVersion type in the module infrastructure.

File: src/Modules/Shared/PowerShellGet/PSModule.psm1 (hunk 2 above)
Comment (bmanikm) [inline src/Modules/Shared/PowerShellGet/PSModule.psm1]: @lzybkr Removed the additional change related to exclusion of .nupkg and .nuspec files in Test-FileCatalog cmdlet usage.

**Comment by quoctruong** [comment]
LGTM

**Comment by raghushantha** [comment]
:shipit:
